opencraft/settings.py

```diff
@@ -916,3 +914,15 @@
 # Batched updates are maximum 500 members at a time, as per
 # https://github.com/VingtCinq/python-mailchimp/blob/ad09dee/mailchimp3/entities/lists.py#L146
 MAILCHIMP_BATCH_SIZE = env.int('MAILCHIMP_BATCH_SIZE', default=500)
+
+# Terms of Service
```

Comment:
Why are ToS and Privacy policy here? They are frontend variables and should be updated only in the CircleCI contexts. Here: https://app.circleci.com/settings/organization/github/open-craft/contexts

Reply:
See: https://github.com/open-craft/opencraft/blob/master/frontend/src/global/constants.ts#L13-L15